### PR TITLE
Bugfix/6.3/tdi 37525 fix delimited issue

### PR DESCRIPTION
**Pull request by jzhao-talend**
Continue to fix the fails tuj.

This also part of the fix.

### CODE REVIEW

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/97/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/97/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/98/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/98/)

**Comment by wwang-talend** [comment]
it's ok.

File: components-filedelimited/src/main/java/org/talend/components/filedelimited/runtime/FileDelimitedAvroRegistry.java [hunk 1]
```diff
@@ -188,7 +186,13 @@ public String convertToDatum(T value) {
 
         @Override
         public Boolean convertToAvro(String value) {
-            return value == null ? null : Boolean.parseBoolean(value);
+            if (value == null) {
+                return null;
+            }
+            if (value.equals("1")) {
+                return Boolean.parseBoolean("true");
```
Comment (jzhao-talend) [inline components-filedelimited/src/main/java/org/talend/components/filedelimited/runtime/FileDelimitedAvroRegistry.java]: yes.

**Pull request merged**